queries.sql

```diff
@@ -1,3 +1,12 @@
 /*Queries that provide answers to the questions from all projects.*/
 
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name <> 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
```

Comment:
- [ ] [Optional] Your query screenshots are a bit disorganized and contain some duplicates. To make the review more straightforward, I recommend organizing your screenshots and removing any duplicates. This will help both you and the reviewers.